apps/src/music/views/MusicView.jsx

```diff
@@ -248,8 +250,12 @@ class UnconnectedMusicView extends React.Component {
     });
   };
 
-  executeSong = () => {
-    this.musicBlocklyWorkspace.executeSong({
+  compileSong = () => {
+    return this.musicBlocklyWorkspace.compileSong();
+  };
```

Comment:
nit: should we just remove this helper and call `this.musicBlocklyWorkspace.compileSong()` directly now that it's just wrapping the call?

Reply:
Fair point, though I kind of like having the calls to `compileSong` and `executeCompiledSong` looking similar to each other.